src/main/java/seedu/address/logic/commands/DeleteAppointment.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+public class DeleteAppointment extends DeleteCommand {
```

Comment:
DeleteAppointmentCommand

Reply:
Note that in the application, the "DeleteCommand" is effectively an "DeletePersonCommand". Please do not be confused and think it is a generic class for deleting items. It is not and thus it should not extend the DeleteCommand (unless it has been explicitly modified to be a generic class, which I doubt so in this case).